support/acks.md

```diff
@@ -0,0 +1,76 @@
+---
+breadcrumb: Acknowledgements
+---
+# Acknowledgements
+
+The OpenSSL project depends on volunteer efforts and financial support
+from the end user community. That support comes in [many
+forms.](donations.html)
+
+## <a href="current">Sponsorship Donations</a>
+
+We would like to identify and thank the following sponsors for their
+donations which give significant support to the OpenSSL project. Please
+note sponsors may choose to remain anonymous.
+
+------------------------------------------------------------------------
+
+<style type="text/css">
+  .sponsorlogo {
+  height: 100px !important;
+  width: 210px !important;
+  object-fit: contain !important;
+  object-position: 50% 50% !important;
+  padding-left: 15px !important;
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+  padding-right: 15px !important;
+  }
+  .sponsorsection {
+  background-color: #ffffff !important;
+  padding-top: 10px !important;
+  text-align: center !important;
+  }
+</style>
+
+### Gold:
+
+<div class="sponsorsection">
+
+[<span class="sponsorlogo">![NGINX](/img/nginx-logo-med.png)</span>](https://www.nginx.com/)
+[<span class="sponsorlogo">![Shiguredo Inc.](/img/shiguredo-logo-med.png)</span>](https://shiguredo.jp/)
```

Comment:
This construction doesn't seem to work right...  the span class doesn't catch on, and this isn't quite the way the original HTML does it (it assigns the class to the img tag), so I'm converting them to use a class attribute, as documented here: https://pandoc.org/MANUAL.html#extension-link_attributes

GFM doesn't render such attributes well, even though it has borrowed other things from PHP Markdown Extra...